go/base/management/commands/go_migrate_conversations.py

```diff
@@ -133,11 +136,10 @@ class SplitBatches(Migration):
         " conversation batches to the new batch.")
 
     def applies_to(self, user_api, conv):
-        mdb = user_api.api.mdb
-        tag_keys = mdb.current_tags.index_keys('current_batch', conv.batch.key)
-        if tag_keys:
-            return True
-        return False
+        batch_manager = user_api.api.get_batch_manager()
+        batch = batch_manager.get_batch(conv.batch.key)
+        current_tags = [batch_manager.get_tag_info(tag) for tag in batch.tags]
+        return any(ct.current_batch.key == batch.key for ct in current_tags)
```

Comment:
I'm guessing we can consider deprecating this code because we no longer have any batches like this?